definitions/npm/fuzzaldrin-plus_v0.6.0/flow_v0.76.x/fuzzaldrin-plus_v0.6.0.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+// Flow-typed type definitions for fuzzaldrin-plus v0.6.0
+// Project: https://github.com/jeancroy/fuzz-aldrin-plus
+// Author: Jean Christophe Roy <https://github.com/jeancroy>
+// Definitions by: Ronald Rey <https://github.com/reyronald>
```

Comment:
You can create a `.md` file in `/npm/fuzzaldrin-plus_v0.6.0` with any info you want future contributors to know. Something like an `AUTHORS.md` or `CONTRIBUTING.md`.